samples/bluetooth/peripheral_uart/prj.conf

```diff
@@ -39,3 +44,12 @@ CONFIG_DK_LIBRARY=y
 
 # This example requires more workqueue stack
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Config logger
+CONFIG_LOG=y
+CONFIG_LOG_MINIMAL=n
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_LOG_PRINTK=n
```

Comment:
This is the default, you shouldn't need to specify it.

Reply:
Right. This is not needed. Removed